src/ic-certified-assets/src/lib.rs

```diff
@@ -693,6 +693,23 @@ fn http_request(req: HttpRequest) -> HttpResponse {
                 encodings.push(v.trim().to_string());
             }
         }
+        if name.eq_ignore_ascii_case("Host") {
+            if let Some(host) = value.split(':').next() {
+                if host.contains("raw.ic0.app") {
+                    let replacement_url = if host.contains("http") {
+                        host.replace("raw.", "") + &req.url
```

Comment:
What is the expected behavior if the Host value is `withdraw.bedstraw.raw.ic0.app` ?